.NET/Microsoft.Recognizers.Definitions.Common/Arabic/ChoiceDefinitions.cs

```diff
@@ -23,7 +23,8 @@ public static class ChoiceDefinitions
     {
       public const string LangMarker = @"Ara";
       public const string TokenizerRegex = @"[^\u0621-\u064A0-9]";
-      public const string TrueRegex = @"\b(صحيح|نعم|حسنا|موافق|متأكد|بالتأكيد|أتفق|اتفق|بالطبع|هيا)\b|(\uD83D\uDC4D|\uD83D\uDC4C|\u0001f44c)";
-      public const string FalseRegex = @"\b(لست متاحا|لا أستطيع|ليس تماما|أختلف|نختلف|اوافق|لست أوافق|لا|ليس صحيح|ليس|ليست|غير موافق)\b|(\uD83D\uDC4E|\u270B|\uD83D\uDD90|\u0001F44E|\u0001F590)";
+      public const string SkinToneRegex = @"(\uD83C\uDFFB|\uD83C\uDFFC|\uD83C\uDFFD|\uD83C\uDFFE|\uD83C\uDFFF)";
```

Comment:
These timexes could be simplified to avoid repetition and it would also help a bit with perf. But impact doesn't seem too big to block merging.